mods/content/sexcon/datum/sexcon.dm

```diff
@@ -0,0 +1,603 @@
+		if(!grabstate)
+			return FALSE
+	return TRUE
+
+/datum/sex_controller/proc/get_speed_multiplier()
+	switch(speed)
+		if(SEX_SPEED_LOW)
+			return 1.0
+		if(SEX_SPEED_MID)
+			return 1.5
+		if(SEX_SPEED_HIGH)
+			return 2.0
+		if(SEX_SPEED_EXTREME)
+			return 2.5
+
+/datum/sex_controller/proc/get_stamina_cost_multiplier()
+	switch(force)
+		if(SEX_FORCE_LOW)
+			return 1.0
+		if(SEX_FORCE_MID)
+			return 1.5
+		if(SEX_FORCE_HIGH)
+			return 2.0
+		if(SEX_SPEED_EXTREME)
+			return 2.5
+
+/datum/sex_controller/proc/get_force_pleasure_multiplier(passed_force, giving)
+	switch(passed_force)
+		if(SEX_FORCE_LOW)
+			if(giving)
+				return 0.8
+			else
+				return 0.8
+		if(SEX_FORCE_MID)
+			if(giving)
+				return 1.2
+			else
+				return 1.2
+		if(SEX_FORCE_HIGH)
+			if(giving)
+				return 1.6
+			else
+				return 1.2
+		if(SEX_FORCE_EXTREME)
+			if(giving)
+				return 2.0
+			else
+				return 0.8
+
+/datum/sex_controller/proc/get_force_pain_multiplier(passed_force)
+	switch(passed_force)
+		if(SEX_FORCE_LOW)
+			return 0.5
+		if(SEX_FORCE_MID)
+			return 1.0
+		if(SEX_FORCE_HIGH)
+			return 2.0
+		if(SEX_FORCE_EXTREME)
+			return 3.0
+
+/datum/sex_controller/proc/get_speed_pain_multiplier(passed_speed)
+	switch(passed_speed)
+		if(SEX_SPEED_LOW)
+			return 0.8
+		if(SEX_SPEED_MID)
+			return 1.0
+		if(SEX_SPEED_HIGH)
+			return 1.2
+		if(SEX_SPEED_EXTREME)
+			return 1.4
+
+/datum/sex_controller/proc/get_force_string()
+	switch(force)
+		if(SEX_FORCE_LOW)
+			return "<font color='#eac8de'>GENTLE</font>"
+		if(SEX_FORCE_MID)
+			return "<font color='#e9a8d1'>FIRM</font>"
+		if(SEX_FORCE_HIGH)
+			return "<font color='#f05ee1'>ROUGH</font>"
+		if(SEX_FORCE_EXTREME)
+			return "<font color='#d146f5'>BRUTAL</font>"
+
+/datum/sex_controller/proc/get_speed_string()
+	switch(speed)
+		if(SEX_SPEED_LOW)
+			return "<font color='#eac8de'>SLOW</font>"
+		if(SEX_SPEED_MID)
+			return "<font color='#e9a8d1'>STEADY</font>"
+		if(SEX_SPEED_HIGH)
+			return "<font color='#f05ee1'>QUICK</font>"
+		if(SEX_SPEED_EXTREME)
+			return "<font color='#d146f5'>UNRELENTING</font>"
+
+/datum/sex_controller/proc/get_generic_force_adjective()
+	switch(force)
+		if(SEX_FORCE_LOW)
+			return pick(list("gently", "carefully", "tenderly", "gingerly", "delicately", "lazingly"))
+		if(SEX_FORCE_MID)
+			return pick(list("firmly", "vigorously", "eagerly", "steadily", "intently"))
+		if(SEX_FORCE_HIGH)
+			return pick(list("roughly", "carelessly", "forcefully", "fervently", "fiercely"))
+		if(SEX_FORCE_EXTREME)
+			return pick(list("brutally", "violently", "relentlessly", "savagely", "mercilessly"))
+
+/datum/sex_controller/proc/spanify_force(string)
+	switch(force)
+		if(SEX_FORCE_LOW)
+			return "<span class='love_low'>[string]</span>"
+		if(SEX_FORCE_MID)
+			return "<span class='love_mid'>[string]</span>"
+		if(SEX_FORCE_HIGH)
+			return "<span class='love_high'>[string]</span>"
+		if(SEX_FORCE_EXTREME)
+			return "<span class='love_extreme'>[string]</span>"
```

Comment:
```suggestion
	var/force_span
	switch(force)
		if(SEX_FORCE_LOW)
			force_span = "love_low"
		if(SEX_FORCE_MID)
			force_span = "love_mid"
		if(SEX_FORCE_HIGH)
			force_span = "love_high"
		if(SEX_FORCE_EXTREME)
			force_span = "love_extreme"
	return SPAN_CLASS(force_span, string)
```
personally i'd prefer to do it this way, although if you add `SPAN_LOVE_LOW` defines those might be more preferable